frontend/locales/en/strings.json

```diff
@@ -27,6 +27,10 @@
         "title": "AWS ParallelCluster Manager documentation",
         "link": "https://pcluster.cloud/"
       }
+    },
+    "default": {
+      "title": "ParallelCluster Manager",
+      "description": "Quickly and easily create HPC cluster in AWS using ParallelCluster Manager. This UI uses the AWS ParallelCluster 3.0 API to Create, Update and Delete Clusters as well as access, view logs, and build Amazon Machine Images (AMI's)."
```

Comment:
issue: Please make sure PC 3.0 is correct (instead of 3.3.0)

Reply:
copy/paste from readme. I will remove the version at all since it's not needed here